src/components/pager/pager.component.tsx

```diff
@@ -47,6 +47,8 @@ export interface PagerProps {
   ) => void;
   /** Current visible page */
   currentPage?: number | string;
+  /** If true, page number navigation will be changed to a non-interactive label */
+  nonInteractivePager?: boolean;
```

Comment:
I think this prop name is a bit vague as it's implying that the whole Pager is not interactive but there will still be buttons/ dropdowns potentially. Maybe something like `hasPageInput` which defaults to true or something like that?

Reply:
I've went with `nonInteractivePageNumber`, let me know what you think 👍

Reply:
Sounds good to me 👍

Reply:
Agreed @edleeks87 - this seems a lot cleaner and clear way of having/using this prop.